### ISSUE TITLE
Pdf reports in node.js not work

### ISSUE THREAD

**Issue by krelfallahhichem** [jsreport/jsreport#159]
after install jsreport with this steps :
sudo npm install jsreport
sudo node node_modules/jsreport --init
sudo node server.js
jsreport run correctly in  https://localhost 

i try this example (from documentation http://jsreport.net/learn/adapting-jsreport )
var http = require('http');
var jsreport = require('jsreport');
http.createServer(function (req, res) {
  jsreport.render("<h1>Hello world</h1>").then(function(out) {
    out.stream.pipe(res);
  }).catch(function(e) {    
    res.end(e.message);
  });
}).listen(1337, '127.0.0.1');
the browser return this error : 
Error during rendering report: Engine 'handlebars' not found
i need help.

**Comment by pofider** [comment]
Thank you. We will take a look at that.

**Comment by krelfallahhichem** [comment]
Hi, i'm waiting your response.

**Comment by pofider** [comment]
Sorry for the late response. I can confirm the rendering shortcut is broken in the 0.10. 

You can one line workaround like this:
```js
var http = require('http');
var jsreport = require('jsreport');
/***** change ******/
jsreport.renderDefaults.rootDirectory = __dirname;

http.createServer(function (req, res) {
  jsreport.render("<h1>Hello world</h1>").then(function(out) {
    out.stream.pipe(res);
  }).catch(function(e) {    
    res.end(e.message);
  });
}).listen(1337, '127.0.0.1');
```

**Comment by krelfallahhichem** [comment]
thank you very match, it's working.

**Comment by pofider** [comment]
Fix included in release 0.11.0
